butterfree/transform/feature_set.py

```diff
@@ -380,6 +382,23 @@ def _filter_duplicated_rows(self, df):
 
         return df.select([column for column in self.columns])
 
+    def filter_dataframe_by_date(
+        self, df, start_date: str = None, end_date: str = None
+    ):
+        """Filter, based on a time period, a given dataframe.
```

Comment:
Why create a new method? Wouldn't it be better to leave the `IncrementalStrategy` instantiation in Agg as it is but just transfers the `self.incremental_strategy` parameter in the init to the Feature set and inherit it as you did? :thinking: :thought_balloon:

Reply:
I've applied these modifications.